src/constants.rs

```diff
@@ -50,6 +50,7 @@ pub const PUBSERVER_BACKUP_DIR: &str = "pubd_bk";
 pub const REPOSITORY_DIR: &str = "repo";
 pub const REPOSITORY_RRDP_DIR: &str = "rrdp";
 pub const REPOSITORY_RRDP_ARCHIVE_DIR: &str = "archive";
+pub const RRDP_FIRST_SERIAL: u64 = 1;
```

Comment:
Would it be worth naming this RRDP_FIRST_SERIAL_PER_RFC_8182 or leaving a comment or something to prevent someone in future thinking they can change this if they wish?

Reply:
I would rather keep it short(-ish). Can add a comment.

Reply:
added